wireless_modules/middle_module/README.md

```diff
@@ -0,0 +1,62 @@
+# Data Acquisition System - Middle wireless module
```

Comment:
`Data Acquisition System - Middle Wireless Module`

- Capital letters for headers.